prov/verbs/configure.m4

```diff
@@ -67,7 +78,8 @@ AC_DEFUN([FI_VERBS_CONFIGURE],[
 
 	#See if we have XRC support
 	VERBS_HAVE_XRC=0
-	AS_IF([test $verbs_ibverbs_happy -eq 1],[
+	AS_IF([test $verbs_ibverbs_happy -eq 1 && \
+	       test $verbs_rdmacm_ex_happy -eq 1],[
 		AC_CHECK_DECL([IBV_QPT_XRC_SEND],
```

Comment:
Is there a single check that we can do for xrc?  If rdma_create_qp_ex() is available, does that indicate that IBV_QPT_XRC_SEND must be present?

Reply:
https://github.com/linux-rdma/rdma-core/commit/995eb0c90c1a0967179fe3f523861e15300d3dfa : tag v22.1
https://github.com/linux-rdma/rdma-core/commit/c7e3e61052dd756c394d8fbccbc498aa4eebbd37 : tag v22.1

Support for XRC Send QPs and that specific API call are part of different commits, but they should exist at the same time given the tag. 

@sydidelot  is observing this problem, which I would guess means that he isn't using rdma-core, but rather ibverbs and librdmacm separately. If that is the case, then I don't know if he has a resolution other than this additional check either installing a newer librdmacm or removing both packages and installing a new rdma-core.

Reply:
@jswaro, thanks for looking into this. Here are the RPMs that are installed on the node that reports the implicit declaration of `rdma_create_qp_ex`:

```
$ rpm -qa | grep ibverbs
libibverbs-devel-static-1.1.7mlnx1-OFED.2.3.124.g4d8b179.x86_64
libibverbs-utils-1.1.7mlnx1-OFED.2.3.124.g4d8b179.x86_64
libibverbs-devel-1.1.7mlnx1-OFED.2.3.124.g4d8b179.x86_64
libibverbs-1.1.7mlnx1-OFED.2.3.124.g4d8b179.x86_64

$ rpm -qa | grep rdmacm 
librdmacm-1.0.17.2mlnx3-OFED.2.3.124.gc0b63e7.x86_64
librdmacm-devel-1.0.17.2mlnx3-OFED.2.3.124.gc0b63e7.x86_64
librdmacm-utils-1.0.17.2mlnx3-OFED.2.3.124.gc0b63e7.x86_64
```

They apparently comes from Mellanox OFED 2.3.124.